spec/features/user_sign_up_spec.rb

```diff
@@ -2,8 +2,7 @@
 
 feature "user can log in" do
   scenario "user fills in details" do
-    visit root_path
-    click_link "Sign Up"
+    visit new_user_registration_path
```

Comment:
I might recommend reverting back, or at least still including something along the lines of `click_link` to ensure that somewhere in the app a user has the ability to sign up!

Reply:
Nick said we can yoink this because our navbar is constructed in React.